src/postgraphile/http/createPostGraphileHttpRequestHandler.ts

```diff
@@ -94,15 +111,14 @@ const origGraphiqlHtml = new Promise((resolve, reject) => {
  * We need to be able to share the withPostGraphileContext logic between HTTP
  * and websockets
  */
-const withPostGraphileContextFromReqResGenerator = options => {
+function withPostGraphileContextFromReqResGenerator(options: ICreateRequestHandler): any {
```

Comment:
Rather than `any`, consider using something like:

```ts
function withPostGraphileContextFromReqResGenerator<T>(options: ICreateRequestHandler): (req: IncomingMessage, res: ServerResponse, moreOptions: any, fn: (pgClient: PoolClient) => T) => T
```

You can use `any` instead of `T` if you like. Effectively whatever is returned by the callback function `fn` will be returned by the function returned from withPostGraphileContextFromReqResGenerator